spec/flatirons/saas/services/stripe_service_spec.rb

```diff
@@ -446,5 +446,18 @@
         end
       end
     end
+
+    describe 'delete subscription' do
+      let!(:subscription) { Stripe::Subscription.create({ customer: customer.id, items: [{ price: price.id }], expand: ['latest_invoice.payment_intent'] }) }
+      before do
+        expect(subscription.status).to eq 'active'
+      end
+
+      it 'should delete the subscription' do
+        deleted_subscription = service.delete_subscription(subscription.id)
+        expect(subscription.id).to eq deleted_subscription.id
+        expect(deleted_subscription.status).to eq 'canceled'
+      end
+    end
```

Comment:
Missing stripe_opts test

```
  describe 'delete_subscription' do
      it 'should raise an error' do
        expect { service.delete_subscription('subscription_id') }.to raise_error 'Stripe API key not configured'
      end
    end
```

Reply:
looks like something that could be into the initialize method of the service, that way each method doesnt need this test and this error can be thrown on StripeService.new